.github/workflows/daily-runtime-validation.yml

```diff
@@ -53,6 +59,7 @@ jobs:
           python3 test-runner/wasi_test_runner.py \
             -r ./adapters/${{ matrix.runtime }}.py \
             --json-output-location results.json \
+            -f ./tests/rust/${{ matrix.runtime }}_skip.json \
```

Comment:
I guess this causes failure until this is synced with `prod/testsuite-base` branch. Could you suggest what I should do?  @loganek